src/device/router_device_worker.erl

```diff
@@ -1387,7 +1402,7 @@ validate_frame(
             validate_frame_(
                 PacketFCnt, Packet, PubKeyBin, Region, Device0, OfferCache, Blockchain, false
             );
-        undefined when FrameAck == 0 andalso PacketFCnt =< LastSeenFCnt ->
+        undefined when FrameAck == 0 andalso PacketFCnt =:= LastSeenFCnt ->
```

Comment:
I don't know if this is right, what if a hotspot sent delivered a "very old" packet?

Reply:
Resolved by the latest series of commits.